dev/update-palette-system-data.js

```diff
@@ -1,8 +1,8 @@
-import puppeteer from 'puppeteer';
+import { launch } from 'puppeteer';
```

Comment:
Ah, I see. Also resolved by `import * as puppeteer from 'puppeteer';` if you prefer. The eslint plugin is trying to protect you from what would plausibly be a bug if puppeteer wasn't doing both exports, though I don't really know why it doesn't just check and only error if it's invalid.